src/libraries/System.Net.Http/src/System.Net.Http.csproj

```diff
@@ -619,7 +619,7 @@
     <Reference Include="System.Collections" />
     <Reference Include="System.Collections.Concurrent" />
     <Reference Include="System.Collections.NonGeneric" />
-    <Reference Include="System.Console" Condition="'$(Configuration)' == 'Debug'" />
+    <Reference Include="System.Console"/>
```

Comment:
Should that change be reverted?

Reply:
Yes all the changes from the last commit are to be reverted. This one included.
All the Console.WriteLines will go away. you can ignore them.